lib/round-tracker.js

```diff
@@ -214,9 +214,10 @@ export async function maybeCreateSparkRound (pgClient, {
 
 async function defineTasksForRound (pgClient, sparkRoundNumber) {
   await pgClient.query(`
-    INSERT INTO retrieval_tasks (round_id, cid, provider_address, protocol)
-    SELECT $1 as round_id, cid, provider_address, protocol
-    FROM retrieval_templates
+    INSERT INTO retrieval_tasks (round_id, cid)
+    SELECT $1 as round_id, cid
+    FROM retrievable_deals
+    WHERE expires_at > now()
```

Comment:
why was this condition added and not necessary before?

Reply:
Great question!

There is no `expires_at` column in `retrieval_templates`.